constrain/checklib.py

```diff
@@ -285,26 +285,27 @@ def day_plot_aio(self, plt_pts, fig_size):
         plotday_filtered = plotday[plotday != "Untested"]
 
         # flag
-        ax1 = plt.subplot(2, 1, 1)
-        sns.scatterplot(x=plotday_filtered.index, y=plotday_filtered)
-        plt.xlim([plotday.index[0], plotday.index[-1]])
-        plt.ylim([-0.2, 1.2])
-        plt.title(f"Example day Pass / Fail flag - {self.__class__.__name__}")
-
-        # datapoints
-        ax2 = plt.subplot(2, 1, 2)
-        plotdaydf[plt_pts].plot(ax=ax2)
-        pt_nan = plotdaydf.isnull().any().to_dict()
-        for i, line in enumerate(ax2.get_lines()):
-            line_label = line.get_label()
-            if pt_nan[line_label]:
-                line.set_marker(".")
-        ax2.ticklabel_format(useOffset=False, axis="y")
-
-        plt.title(f"Example day data points plot - {self.__class__.__name__}")
-        plt.tight_layout()
-        plt.savefig(f"{self.results_folder}/Day_plot_aio.png")
-        print()
+        if plotday_filtered.size != 0:
```

Comment:
@leijerry888 The reason why I added this condition is to prevent an error when the size of the ```plotday_filtered``` is 0,

Reply:
![Uploading image.png…]()

Reply:
Based on the discussion, we decided not to make updates for plotting in this PR (Interactive plot PR will handle the issue).